### PR TITLE
Feature create bill

### PR DESCRIPTION
**Pull request by at-dungnguyen2**
hi a Dung, i have fixed your comments, please help me review and merge . 
thank you in advanced

### CODE REVIEW

File: public/js/bills/main.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
+var products;
+var old_cost = 0;
+var current_cost = 0;
+var index = 1;
+$(document).ready(function(){
+  $.getJSON( url, function( data ) {
+    products = data;
+    setAutocomplete('#item' + index + ' .product', '#item' + index + ' .product_id', '#item' + index + ' .price');
```
Comment (dzungnguyenh) [inline public/js/bills/main.js]: [rank C] please separate and declare variables for these.

File: public/js/bills/main.js [hunk 2]
```diff
@@ -0,0 +1,73 @@
...
+  index++;
+});
+$(document).on('click', '.btn_remove', function(){
+  $(this).parent().parent().remove();  
+});
+$(document).on('keyup click', '.amount-box', function () {
+  var item = $(this).parent().parent();
+  var amount = $(item).find('.amount-box').val();
+  var price = $(item).find('.price-box').val();
+  current_cost = old_cost+Number(price)*(amount-1);
```
Comment (at-dungnh) [inline public/js/bills/main.js]: [rank B] Please pay attention for variable's scope.

File: public/js/bills/main.js [hunk 3]
```diff
@@ -0,0 +1,73 @@
...
+  $('#total_cost').val(current_cost);
+  $('#cost_display').val($('#total_cost').val() + currency_label);
```
Comment (dzungnguyenh) [inline public/js/bills/main.js]: [rank C] Declare variable for this

File: resources/views/bills/create.blade.php [hunk 4]
```diff
@@ -0,0 +1,116 @@
...
+        <div class="row">
+          <div class="form-group">
+            <label for="description">@lang('bills.create.description_label'): </label>
+            <textarea class="form-control" name="description"></textarea>
+          </div>
+        </div>
+        <div class="row">
+          <div class="form-group">
+            <div class="col-md-3 col-sm-3 col-xs-12 col-md-offset-3">
+              <a href="{{ route('bill.create') }}" class="btn btn-danger"><i class="fa fa-refresh"></i> Reset</a>
```
Comment (dzungnguyenh) [inline resources/views/bills/create.blade.php]: [rank C] I don't want see something like that in future 👎

**Comment by at-dungnguyen2** [comment]
@at-dungnh  i have fixed your comments, please help me review and merge . 
thank you in advanced

**Comment by dzungnguyenh** [comment]
👍 So good.

**Pull request merged**